.rubocop.yml

```diff
@@ -1,3 +1,6 @@
+AllCops:
```

Comment:
Maybe we can add these in the configuration. I don't really have a reference for this but most of the time when Im adding rubocop to other projects I always include these just to be more practical haha

```
Metrics/MethodLength:
  Max: 30

Metrics/ModuleLength:
  Max: 150

Metrics/AbcSize:
  Max: 35

Style/ModuleFunction:
  EnforcedStyle: extend_self

Style/Documentation:
  Enabled: false
```

Reply:
I like those, other than:

```
Style/Documentation:
  Enabled: false
```

...mainly because I'd like to make sure the gems are documented.

Reply:
Yeah agree that cop is for adding comments at the first line of any class or module let's leave that one

Reply:
I'll update and merge ASAP.